lte/gateway/c/core/oai/tasks/ngap/ngap_amf_handlers.c

```diff
@@ -1551,14 +1550,14 @@ status_code_e ngap_handle_sctp_disconnection(ngap_state_t* state,
       &gnb_association->ue_id_coll, ngap_send_gnb_deregistered_ind, (void*)&arg,
       (void**)&message_p);
 
-  for (i = arg.current_ue_index; i < NGAP_ITTI_UE_PER_DEREGISTER_MESSAGE; i++) {
-    NGAP_GNB_DEREGISTERED_IND(message_p).amf_ue_ngap_id[arg.current_ue_index] =
-        0;
-    NGAP_GNB_DEREGISTERED_IND(message_p).gnb_ue_ngap_id[arg.current_ue_index] =
-        0;
-  }
-  NGAP_GNB_DEREGISTERED_IND(message_p).gnb_id = gnb_association->gnb_id;
-  message_p = NULL;
+  /*
+   * Mark the eNB's s1 state as appropriate, the eNB will be deleted or
```

Comment:
Change the comment to gNB at all places wherever it is relevant

Reply:
Addressed Review Comments